test/test_docling_doc.py

```diff
@@ -1248,3 +1257,99 @@ def test_save_to_disk():
     _verify_saved_output(filename=filename, paths=paths)
 
     assert True
+
+
+def test_document_manipulation():
```

Comment:
Let's also cover adding a sibling to the root (body), which should raise an exception.